templates/chatbot_ui.html

```diff
@@ -9,13 +9,17 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet">
-    <link href="assets/css/chatbot_ui.css" rel="stylesheet">
-
-    <!-- Optional JavaScript -->
+    <!-- <link href="../static/styles/chatbot_ui.css" rel="stylesheet"> -->
```

Comment:
IIRC Flask has a special template tag for static files - might be worth using that instead?

Reply:
I had issues with that tbh. Couldn't get it to load the css properly. Perhaps you could do it?